Src/Extern/HYPRE/AMReX_HypreABecLap.cpp

```diff
@@ -195,24 +200,16 @@ HypreABecLap::prepareSolver ()
     std::iota(stencil_indices.begin(), stencil_indices.end(), 0);
     const Real* dx = geom.CellSize();
     const int bho = (m_maxorder > 2) ? 1 : 0;
-    FArrayBox rfab;
+    BaseFab<GpuArray<Real,regular_stencil_size>> rfab;
     for (MFIter mfi(acoefs); mfi.isValid(); ++mfi)
     {  
         const Box &reg = mfi.validbox();
 
         rfab.resize(reg,regular_stencil_size);
```

Comment:
```suggestion
        rfab.resize(reg);
```
because each element of the fab is already an array.